app/controllers/observations_controller/index.rb

```diff
@@ -1,223 +1,232 @@
+      show_selected_observations(query)
+    end
 
-  # Displays matrix of Observations with the given text_name (or search_name).
-  def observations_of_name
-    query = create_query(:Observation, :all,
-                         names: [params[:name]],
-                         include_synonyms: true,
-                         by: :confidence)
-    show_selected_observations(query)
-  end
+    def default_sort_order
+      ::Query::ObservationBase.default_order
+    end
 
-  # Displays matrix of Observations of subtaxa of the parent of the given name.
-  def observations_of_related_taxa
-    query = create_query(:Observation, :all,
-                         names: parents(params[:name]),
-                         include_subtaxa: true,
-                         by: :confidence)
-    show_selected_observations(query)
-  end
+    # Displays matrix of selected Observations (based on current Query).
+    # NOTE: Why are all the :id params converted .to_s below?
+    def index_query_results
+      query = find_or_create_query(:Observation, by: params[:by])
+      show_selected_observations(
+        query, id: params[:id].to_s, always_index: true
+      )
+    end
 
-  # Displays matrix of User's Observations, by date.
-  def observations_by_user
-    return unless (
-      user = find_or_goto_index(User, params[:user])
-    )
+    # Displays matrix of Observations with the given name proposed but not
+    # actually that name.
+    def look_alikes
+      query = create_query(:Observation, :all,
+                           names: [params[:name]],
+                           include_synonyms: true,
+                           include_all_name_proposals: true,
+                           exclude_consensus: true,
+                           by: :confidence)
+      show_selected_observations(query)
+    end
 
-    query = create_query(:Observation, :by_user, user: user)
-    show_selected_observations(query)
-  end
+    # Displays matrix of Observations with the given text_name (or search_name).
+    def name
+      query = create_query(:Observation, :all,
+                           names: [params[:name]],
+                           include_synonyms: true,
+                           by: :confidence)
+      show_selected_observations(query)
+    end
 
-  # Displays matrix of Observations at a Location, by date.
-  def observations_at_location
-    return unless (
-      location = find_or_goto_index(Location, params[:location].to_s)
-    )
+    # Displays matrix of Observations of subtaxa of the parent of given name.
+    def related_taxa
+      query = create_query(:Observation, :all,
+                           names: parents(params[:name]),
+                           include_subtaxa: true,
+                           by: :confidence)
+      show_selected_observations(query)
+    end
 
-    query = create_query(:Observation, :at_location, location: location)
-    show_selected_observations(query)
-  end
+    def parents(name_str)
```

Comment:
Does method belong in Name?
(There is a "Name#parents", but it's based on `classification`.)